falcon/__init__.py

```diff
@@ -37,7 +37,13 @@
 )
 
 DEFAULT_MEDIA_TYPE = 'application/json; charset=UTF-8'
-
+HTML = 'text/html; charset=utf-8'
+JSON = DEFAULT_MEDIA_TYPE
+JS = 'text/javascript; charset=utf-8'
+XML = 'text/javascript; charset=utf-8'
```

Comment:
The RFCs generally recommend not specifying a charset for XML unless the XML encoding declaration is known (i.e., `<?xml version="1.0" encoding="UTF-8"?>`) and matches the charset parameter. Therefore, in this case, it is probably best to leave off the `charset` parameter.